twrp-functions.cpp

```diff
@@ -103,10 +103,17 @@ int TWFunc::Exec_Cmd(const string& cmd, bool Show_Errors) {
 			break;
 		default:
 		{
-			if (TWFunc::Wait_For_Child(pid, &status, cmd, Show_Errors) != 0)
-				return -1;
-			else
-				return 0;
+			int ret = TWFunc::Wait_For_Child(pid, &status, cmd, Show_Errors, retn);
+
+			if (retn) {
+				return ret;
+			} else {
```

Comment:
This else condition has no significance since we are returning :
```
if (retn)
    return ret;

if (ret != 0)
    return -1;
else
    return 0;
```

This also improves readability